examples/website/texture-tester/textures-data.js

```diff
@@ -0,0 +1,62 @@
+export const IMAGES_DATA = [
```

Comment:
+1 This programmatic approach makes it much easier for users to understand this example